test/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '2.0'
+
```

Comment:
It'd be nice if we could templatize & generate this file since it's repetitive. If you're up for it [gomplate](https://docs.gomplate.ca/) would be an easy way to do it, but I'm also fine with you merging this as-is.

Reply:
There are a few things that are hard to predict and could make template generation tricky, like the Docker image name and whether to use apt/yum/zypper. There is some repetitive copy-pasta here (and also in other files in this repo), but I think it's okay for now since it only takes a few seconds to delete or add a new section.